packages/core/src/lib/markdown-it/index.js

```diff
@@ -87,7 +87,7 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
     // counter is incremented on each span, so we need to subtract 1
     token.attrJoin('style', `counter-reset: line ${startFromZeroBased};`);
   }
-  
+
```

Comment:
let's keep unrelated changes to another PR (our custom renderers should probably be moved to entirely separate linted file)

Reply:
Are you referring to the whitespace changes here? Looks like WebStorm fixed it out automatically for some reason. Let's see if I can re-add these back

Reply:
yup